terraform/environments/mlra/alb.tf

```diff
@@ -40,4 +48,85 @@ module "alb" {
   stickiness_enabled         = true
   stickiness_type            = "lb_cookie"
   stickiness_cookie_duration = 10800
+
```

Comment:
This is part of https://dsdmoj.atlassian.net/browse/LAWS-3076 adding CloudFront distribution and WAF